plugins/woocommerce/src/Admin/Features/Settings/Init.php

```diff
@@ -53,6 +54,20 @@ public function is_settings_page() {
 		return $screen && 'woocommerce_page_wc-settings' === $screen->id;
 	}
 
+	/**
+	 * Turn off the embed page if the current page is the WooCommerce settings page.
+	 *
+	 * @param bool $is_connected_page Whether the current page is a connected page.
+	 * @return bool
+	 */
+	public function disable_embed_page_for_settings( $is_connected_page ) {
```

Comment:
```suggestion
	public function disable_embed_if_woocommerce_settings_page( $is_connected_page ) {
```
Naming nitpick again 😅